modules/core/src/main/java/org/apache/ignite/internal/IgniteKernal.java

```diff
@@ -3795,7 +3795,7 @@ private void checkNearCacheStarted(IgniteCacheProxy<?, ?> cache) throws IgniteCh
 
     /** {@inheritDoc} */
     @Override public void active(boolean active) {
-        cluster().active(active);
+        cluster().state(active ? ClusterState.ACTIVE : ClusterState.INACTIVE);
```

Comment:
Let's withhold this change.  Both Ignite.active() and IgniteCluster.active() are deprecated. So it's OK for one to map to the former. If there's a warning inside IgniteCluster.active(), it's even better that way.